FDA/FDA/Scenes/PlaceDetail/PlaceDetailViewState.swift

```diff
@@ -26,4 +28,13 @@ struct PlaceDetailViewState: DynamicProperty {
     var first: Place? {
         Places.mock.places.first
     }
+    
+    var isFavourite: Binding<Bool> {
+        .init {
+            placesObject.favouritePlaces?.contains(place.properties.ogcFid) ?? false
+        } set: {
+            newValue in placesObject.set(place: self.place, favourite: newValue);
+        }
+    }
+    // why is this a legit syntax? I don't get it, what means `newValue in`, where newValue is a local var already in use in set(place, favourite) call
```

Comment:
```suggestion
        } set: { newValue in
           placesObject.set(place: self.place, favourite: newValue);
        }
    }
    // why is this a legit syntax? I don't get it, what means `newValue in`, where newValue is a local var already in use in set(place, favourite) call
```

Vaše syntaxe je správná, jenom tím, že jste to měl zvláštně odřádkované to asi není úplně vidět: `newValue` je vaše pojmenování pro argument, který closure přijímá (název udáváte před slovíčkem `in`). Můtete si ho změnit, to je na vás. Ale vždy musí vaše funkce přijímat právě jeden argument. Viz https://docs.swift.org/swift-book/documentation/the-swift-programming-language/closures/